app/views/search.mustache

```diff
@@ -7,7 +7,7 @@
 {{#resourceList}}
   <div class="search-result" data-test="resource">
     <h2 class="search-result-title" data-test="title">
-      <a href="#{{ resourceId }}">{{ title }}</a>
+      <a href="/resources/{{ resourceId }}/detail" data-test="href">{{ title }}</a>
```

Comment:
data-test="href" ... this is a html tag, you could describe the item better so your test code will be clearer
e.g.
data-test="search-result-link"